pkgs/development/python-modules/audio-metadata/default.nix

```diff
@@ -20,7 +20,8 @@ buildPythonPackage rec {
     substituteInPlace setup.py \
       --replace "bidict>=0.17,<0.18" "bidict" \
       --replace "more-itertools>=4.0,<8.0" "more-itertools" \
-      --replace "pendulum>=2.0,<=3.0,!=2.0.5,!=2.1.0" "pendulum>=2.0,<=3.0"
+      --replace "pendulum>=2.0,<=3.0,!=2.0.5,!=2.1.0" "pendulum>=2.0,<=3.0" \
```

Comment:
Did you check if this and the above still apply?

Reply:
No longer relevant.